boards/st/nucleo_l452re/Kconfig.nucleo_l452re

```diff
@@ -3,3 +3,6 @@
 
 config BOARD_NUCLEO_L452RE
 	select SOC_STM32L452XX
+
+config BOARD_NUCLEO_L452RE_STM32L452XX_P
+	select SOC_STM32L452XX
```

Comment:
Does this work ?
From what I've seen in other cases, it was required to indicate symbol type (bool)

Reply:
Done

Reply:
Late finding: `	select SOC_STM32L452XX` should be removed.
From what I understand `BOARD_NUCLEO_L452RE` is set to y for both configurations.
Note that `BOARD_NUCLEO_L452RE_STM32L452XX_P` may not be needed at all if not used. For instance in no Kconfig difference exist.

Reply:
Done